client-course-schedulizer/src/readCSV.ts

```diff
@@ -0,0 +1,428 @@
+import papa from "papaparse";
+import * as di from "./dataInterfaces";
```

Comment:
Just so the `src/` folder doesn't get too cluttered, perhaps you can move the interfaces to `utilities/interfaces` or to another folder of your choosing.

Reply:
Also, I would put `readCSV.ts` somewhere in `utilities/`

Reply:
85a0bbd